src/main/java/javaslang/collection/CharSeq.java

```diff
@@ -386,6 +386,15 @@ public CharSeq intersperse(Character element) {
         return result;
     }
 
+    @Override
+    public CharSeq padTo(int length, Character element) {
+        final StringBuilder sb = new StringBuilder(back);
+        for (int i = 0; i < length; i++) {
```

Comment:
or is it `i < length - back.length()`?

`"12345".padTo(10, 'a')` should be `12345aaaaa`